neurox/interpretation/linear_probe.py

```diff
@@ -7,9 +7,11 @@
         `Dalvi, Fahim, et al. "What is one grain of sand in the desert? analyzing individual neurons in deep nlp models." Proceedings of the AAAI Conference on Artificial Intelligence. Vol. 33. No. 01. 2019. <https://ojs.aaai.org/index.php/AAAI/article/view/4592>`_
 """
 import numpy as np
+import time
 import torch
 import torch.nn as nn
 from torch.autograd import Variable
+from group_lasso import LogisticGroupLasso
```

Comment:
Please add this lib to `requirements.txt` along with a version that you know works well